File: SoftUni Lessons/Python Development/Python Basics April 2019/Lessons and Problems/07.1 - More Exercises Nested Conditional Statements/05. Vacation .py
budget = float(input())
season = input()

accommodation = None
location = None
price = 0

if budget <= 1000:
    accommodation = "Camp"
    if season == "Summer":
        location = "Alaska"
        price = budget * 0.65
    elif season == "Winter":
        location = "Morocco"
        price = budget * 0.45
elif 1000 < budget <= 3000:
    accommodation = "Hut"
    if season == "Summer":
        location = "Alaska"
        price = budget * 0.80
    elif season == "Winter":
        location = "Morocco"
        price = budget * 0.60
elif budget > 3000:
    accommodation = "Hotel"
    price = budget * 0.9
    if season == "Summer":
        location = "Alaska"
    elif season == "Winter":
        location = "Morocco"

print(f"{location} - {accommodation} - {price:.2f}")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/10. УПРАЖНЕНИЕ ПО-СЛОЖНИ ПРОВЕРКИ/Nested Conditional Statements - More Exercises/05. Vacation.js
function vacantion(...input) {
    let budget = Number(input.shift());
    let season = input.shift();  // Winter or Summer

    let accommodation = undefined;  // Hotel, Hut or Camp
    let location = undefined;  // Alaska or Morocco
    let price = undefined;  // Only god knows

    if (budget <= 1000) {
        accommodation = "Camp";
        switch (season) {
            case "Summer":
                location = "Alaska";
                price = budget * 0.65;
                break;
            case "Winter":
                location = "Morocco";
                price = budget * 0.45;
        }
    } else if (budget > 1000 && budget <= 3000) {
        accommodation = "Hut";
        switch (season) {
            case "Summer":
                location = "Alaska";
                price = budget * 0.80;
                break;
            case "Winter":
                location = "Morocco";
                price = budget * 0.60;
        }
    } else {
        accommodation = "Hotel";
        price = budget * 0.9;
        switch (season) {
            case "Summer":
                location = "Alaska";
                break;
            case "Winter":
                location = "Morocco";
        }
    }

    console.log(`${location} - ${accommodation} - ${price.toFixed(2)}`);
}


vacantion(['800', 'Summer']);  // Expected Output:
// Alaska - Camp - 520.00

vacantion(['799.50', 'Winter']);  // Expected Output:
// Morocco - Camp - 359.78

vacantion(['3460', 'Summer']);  // Expected Output:
// Alaska - Hotel - 3114.00

vacantion(['1100', 'Summer']);  // Expected Output:
// Alaska - Hut - 880.00

vacantion(['5000', 'Winter']);  // Expected Output:
// Morocco - Hotel - 4500.00

vacantion(['2543.99', 'Winter']);  // Expected Output:
// Morocco - Hut - 1526.39